script/elastic/up.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# https://hub.docker.com/_/elasticsearch/
+
+
+sysctl -w vm.max_map_count=262144
```

Comment:
yes, sudo is required.
elasticsearch checks max map count. without this, it won't boot up correctly.
https://www.elastic.co/guide/en/elasticsearch/reference/5.0/_maximum_map_count_check.html